examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -162,6 +165,8 @@ extern "C" void app_main()
     }
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI
 
+    chip::Inet::UDPEndPointImplLwIP::SetQueueFilter(&espfilter);
```

Comment:
We had added this to example for now and in future we planned to move it to platform

Reply:
Packet filter rules **HAVE to be by product**, otherwise the filtering is usually too strict and based on the lowest common denominator or on rules that may not apply to all products, and cause really hard to diagnose problems. Packet filters have to be well-documented and **all** the assumptions about them have to be stated.